data/datafiles.json

```diff
@@ -619,8 +619,14 @@
 	    "filename": "ramses_extra_fields",
 	    "size": "220 MB",
 	    "url": "http://yt-project.org/data/ramses_extra_fields.tar.gz"
+	},
+	{
+	    "code": "RAMSES",
+	    "description": "Hydro run, unzip to 26MB with extra particle fields (temperature + family)",
+	    "filename": "ramses_extra_fields_small",
+	    "size": "15 MB",
+	    "url": "http://yt-project.org/data/ramses_custom_fields.tar.gz"
```

Comment:
The `filename` field should match the actual name of the file. Want me to rename it on the webserver? Or you could just make the `filename` be `ramses_custom_fields`.